Lib/test/test_tkinter/test_widgets.py

```diff
@@ -1402,10 +1402,15 @@ def test_configure_title(self):
 
     def test_configure_type(self):
         widget = self.create()
+        r = tkinter.Tk()
+        tkver = r.getvar('tk_version')
+        r.destroy()
+        opts = ('normal, tearoff, or menubar' if tkver < '8.7' else
+                'menubar, normal, or tearoff')
```

Comment:
Just be aware that under the default behavior of `tkinter.wantobjects == 1`, it is not guaranteed that `getvar('tk_version')` will return a string rather than a float; see #38396.

Reply:
So I applied Serhiy's suggestion instead.